docs/en/modules/admin/pages/spaces/processes/types.adoc

```diff
@@ -0,0 +1,38 @@
+= Process types
+
+In some cases it is necessary to organize participatory processes by more than just xref:spaces/processes/groups.adoc[Process Groups]. This is where Process Types can be useful.
+
+Process Types work similarly to xref:assemblies:assemblies.adoc#_assemblies_types[Assembly types].
```

Comment:
The xref are not intuitive for newcomers, at least for me were difficult to grok at the beginning. Most of the times [I go directly to the `nav.adoc` to copy and paste the reference](https://github.com/decidim/documentation/blob/8aba42fb91d31d7e18225ecfd6ba3524f907e904/docs/en/modules/ROOT/nav.adoc?plain=1#L68)

```suggestion
Process Types work similarly to xref:admin:spaces/assemblies.adoc#_assemblies_types[Assembly types].
```

This does the trick